tests/components/stream/test_worker.py

```diff
@@ -729,24 +729,31 @@ def blocking_open(stream_source, *args, **kwargs):
 
 async def test_worker_log(hass, caplog):
     """Test that the worker logs the url without username and password."""
-    stream = Stream(
-        hass,
-        "https://abcd:efgh@foo.bar",
-        {},
-        hass.data[DOMAIN][ATTR_SETTINGS],
-        dynamic_stream_settings(),
-    )
-    stream.add_provider(HLS_PROVIDER)
+    for stream_url, redacted_url in (
```

Comment:
this seems like a good use of `@pytest.mark.parametrize` to set the `stream_url` and `redacted_url` as a test argument

Reply:
I suggest also adding parameters that you don't want redacted, just to make sure...